src/python/dxpy/bindings/dxapplet.py

```diff
@@ -158,7 +161,7 @@ def _get_cleanup_keys(self):
     def run(self, executable_input, project=None, folder=None, name=None, tags=None, properties=None, details=None,
             instance_type=None, stage_instance_types=None, stage_folders=None, rerun_stages=None, cluster_spec=None,
             depends_on=None, allow_ssh=None, debug=None, delay_workspace_destruction=None, priority=None,
-            ignore_reuse=None, ignore_reuse_stages=None, extra_args=None, **kwargs):
+            ignore_reuse=None, ignore_reuse_stages=None, extra_args=None, detach=None, **kwargs):
```

Comment:
Minor: we tend to keep the extra_args=None, **kwargs at the end of the params list, so please move detach=None before extra_args, thanks!

Reply:
fixed